contracts/utils/structs/EnumerableMap.sol

```diff
@@ -57,6 +57,11 @@ library EnumerableMap {
     // This means that we can only create new EnumerableMaps for types that fit
     // in bytes32.
 
+    /**
+     * @dev Query for an inexistent map key.
+     */
+    error EnumerableMapInexistentKey(bytes32 key);
```

Comment:
I'm fine with any of both as long as we keep consistency. Since the EIP-6093 includes `Inexistent` we'll require updating that one as well so I'll keep this open as a stretch goal if we have time to also update the EIP-6093

Reply:
Yeah I see now that we used Inexistent in EIP-6093, I didn't notice that then. I agree they'd have to be consistent.

Reply:
[Updated the EIP](https://github.com/ethereum/EIPs/pull/7133) and also sent a commit replacing `Inexistent` with `Nonexistent`